### PR TITLE
Add BoxTextured and sort deployable files.

### PR DESCRIPTION
**Pull request by diegoteran**
Adds missing deployable file for https://github.com/google/model-viewer/pull/4209

### CODE REVIEW

**Review by elalish — DISMISSED** [review]
Thanks!

File: packages/render-fidelity-tools/test/goldens/khronos-BoxVertexColors/model-viewer-golden.png
Comment (elalish) [inline packages/render-fidelity-tools/test/goldens/khronos-BoxVertexColors/model-viewer-golden.png:1]: Here's a great example of one where the camera should get changed. Just go to `render-fidelity-tools/test/config.json` and tweak the corresponding entry.

File: packages/render-fidelity-tools/test/goldens/khronos-BoxVertexColors/three-gpu-pathtracer-golden.png
Comment (elalish) [inline packages/render-fidelity-tools/test/goldens/khronos-BoxVertexColors/three-gpu-pathtracer-golden.png:1]: @gkjohnson Any thoughts on why the path-tracer vertex colors are so different? I feel like our approximations should be pretty close in this simple case.

**Review by elalish — APPROVED** [review]

File: packages/render-fidelity-tools/test/goldens/khronos-BoxVertexColors/three-gpu-pathtracer-golden.png
Comment (gkjohnson) [reply packages/render-fidelity-tools/test/goldens/khronos-BoxVertexColors/three-gpu-pathtracer-golden.png:1]: Thanks I'll have to take a closer look. I put the pathtracer down for a bit but I'll hopefully be picking it back up soon and I can dive into this.

**Pull request merged**